terraform/table_terraform_resource.go

```diff
@@ -281,3 +348,22 @@ func convertModelDocumentToMapInterface(data interface{}) map[string]interface{}
 	}
 	return result
 }
+
+func removeKicsLabels(data interface{}) interface{} {
```

Comment:
What's an example of the kics label we're removing in the returned parsed data?

Reply:
Below is a sample of one instance with the kics label, which we remove here.
```
"map[attributes:map[_kics_lines:map[_kics__default:{154 []} _kics_dns_suffix:{155 []} _kics_id:{89 []} _kics_partition:{157 []} _kics_reverse_dns_prefix:{158 []}] dns_suffix:amazonaws.com id:aws partition:aws reverse_dns_prefix:com.amazonaws] schema_version:0 sensitive_attributes:[]]"
```